src/pages/RFOX/RFOX.tsx

```diff
@@ -22,7 +22,7 @@ export const RFOX: React.FC = () => (
       <Stack alignItems='flex-start' spacing={4} mx='auto' direction={direction}>
         <Stack spacing={4} flex='1 1 0%' width='full'>
           <Overview />
-          <TxHistory />
+          <History />
```

Comment:
naming improved (this is not purely *Tx* history) and removed dupe and non-working "Claims" from the history component after discussions with product